org-roam-capture.el

```diff
@@ -303,6 +301,20 @@ This function is used solely in Org-roam's capture templates: see
         (push val converted)))
     (append (nreverse converted) `(:org-roam ,org-roam-plist))))
 
+(defun org-roam-capture--run-after-find-file-hook ()
+  "Run the hooks defined in `org-roam-capture-after-finalize-hook'.
+This is added as a hook to `org-capture-finalize-hook'."
+  (unless org-note-abort
+    (org-roam-capture--find-file-h)
+    (run-hooks 'org-roam-capture-after-find-file-hook))
+  (remove-hook 'org-capture-after-finalize-hook #'org-roam-capture--run-after-find-file-hook))
```

Comment:
No need to introduce a new hook here. Substitute line 308 with the `when-let` clause in the previous hook.

Reply:
Let me wrap my head around your suggestion, and I'll make the modification.

Reply:
Did you mean something like in 22d96fe?
https://github.com/jethrokuan/org-roam/blob/22d96fe338bf6607b80d875eed0db48b00d935ac/org-roam-capture.el#L298-L305

Reply:
I've removed `org-roam-capture--run-after-find-file-hook` from the code, but there was another mention of it here (L34):
https://github.com/jethrokuan/org-roam/blob/ba80a31fe0ddb48d41c446560c35c5677214a6ce/org-roam-dailies.el#L49-L55

Not sure if this has an impact, since I don't use that functionality.